run.sh

```diff
@@ -0,0 +1,3 @@
+export FLASK_APP=views.py
```

Comment:
That's so the `flask` command knows where you've put the application instance ([docs](http://flask.pocoo.org/docs/0.12/quickstart/#a-minimal-application)).